client/src/context.js

```diff
@@ -24,16 +25,21 @@ export class AuthProvider extends React.Component {
 
   render() {
     const { logged, admin } = this.state;
+    const { children } = this.props;
     return (
       <Context.Provider
         value={{
           logged,
           admin,
+          setAdmin: (Admin) => this.setState({ admin: Admin }),
+          setLoged: (Loged) => this.setState({ logged: Loged }),
```

Comment:
```suggestion
          setAuth: ({ logged, admin }) => setState({ logged, admin })
```
make it something like this, it looks better and may help us later on

Reply:
for example u need to call setAdmin and then setLoged
unneeded redundancy